docs/content/en/docs/osmgmt/artifacts.md

```diff
@@ -922,14 +932,75 @@ These steps use `image-builder` to create a Ubuntu-based image for Nutanix AHV a
 
       * `--os`: `ubuntu`
       * `--hypervisor`: For Nutanix use `nutanix`
-      * `--release-channel`: Supported EKS Distro releases include 1-23, 1-24 1-25 1-26 and 1-27
+      * `--release-channel`: Supported EKS Distro releases include 1-23, 1-24, 1-25, 1-26 and 1-27.
       * `--nutanix-config`: Nutanix configuration file (`nutanix-connection.json` in this example)
 
       ```bash
       cd /home/$USER
       image-builder build --os ubuntu --hypervisor nutanix --release-channel 1-27 --nutanix-config nutanix-connection.json
       ```
 
+
+### Configuring OS version
+
+`image-builder` supports an `os-version` flag that allows you to configure which version of the OS you wish to build. If no OS version is supplied, it will build the default for that OS, according to the table below.
+
+<table style>
+    <thead align="center">
+        <tr>
+            <th><center>Operating system</th>
+            <th><center>Supported versions</th>
```

Comment:
The supported versions here is tricky. We do support these versions but we should probably also add a line saying image-builder will pick the latest version on that release train by running upgrade, so that users know if they choose --os-version 2004, they dont always get 20.04.1. This is the same for Rhel too. Maybe we can replicate what we did for RHEL there with a `+` and add a note on what that means.

Reply:
Good call! Will add a note and the `+`.